### PR TITLE
chore: add liveness swift api documentation

### PR DESCRIPTION
**Pull request by phantumcode**
#### Description of changes
* Add documentation for Liveness Swift UI parameters and error states.

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/liveness/full-api.swift.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### FaceLivenessDetectorView Parameters
+
+Below is the full list of parameters that can be used with the `FaceLivenessDetectorView` component. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui-swift-liveness/blob/main/Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionView.swift#L27).
+
+<ReactPropsTable props={FACE_LIVENESS_DETECTOR_PARAMETERS} />
+
+### Error States
+
+Below is the full list of error states that can be returned from the `onCompletion` handler of `FaceLiveNessDetectorView`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui-swift-liveness/blob/main/Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionError.swift).
```
Comment (tjleing) [inline docs/src/pages/[platform]/connected-components/liveness/full-api.swift.mdx:15]:
```suggestion
Below is the full list of error states that can be returned from the `onCompletion` handler of `FaceLivenessDetectorView`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui-swift-liveness/blob/main/Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionError.swift).
```

File: docs/src/pages/[platform]/connected-components/liveness/swift-props.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  },
+  {
+    name: `.sessionNotFound`,
+    description: 'Session not found.',
+    type: `Error`,
+  },
+  {
+    name: `.sessionTimedOut`,
+    description:
+      'The session timed out and id not receive response from server within the time limit.',
```
Comment (tjleing) [inline docs/src/pages/[platform]/connected-components/liveness/swift-props.ts:52]:
```suggestion
      'The session timed out and did not receive response from server within the time limit.',
```

**Comment by reesscot** [comment]
Swift:
https://phantum-swift-liveness-doc-update.d22cn4asamk913.amplifyapp.com/swift/connected-components/liveness#full-api-reference

Android:
https://phantum-swift-liveness-doc-update.d22cn4asamk913.amplifyapp.com/android/connected-components/liveness#full-api-reference

**Review by reesscot — APPROVED** [review]
LGTM

**Review by calebpollman — APPROVED** [review]

**Review by atierian — APPROVED** [review]

**Pull request merged**